py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/configuration.py

```diff
@@ -1,69 +0,0 @@
-import pyarrow
-
-from .config_base import ConfigType, ConfigDetail
-from .config_rt_alerts import RtAlertsDetail
-from .config_rt_trip import RtTripDetail
-from .config_rt_vehicle import RtVehicleDetail
-from .config_rt_bus_vehicle import RtBusVehicleDetail
-from .config_rt_bus_trip import RtBusTripDetail
-from .error import NoImplException
-
-
-class Configuration:
```

Comment:
This class has become the `Converter` abstract base class.

Its children will handle all conversions from a specific input filetype to parquet tables paired with s3 prefixes.